### ISSUE TITLE
How to add hyperlink?

### ISSUE THREAD

**Issue by HelenDias** [linways/table-to-excel#1]
Hi!
Anyone can help me?
I have a HTML with my table and I'm using style classes in td and/or tr. I need to add a hyperlink on cell, but I don't find how to do that. 
Also, I will like to add more than one hyperlink at the same cell (td).
Anyone have any example to help me?

**Comment by rohithb** [comment]
Instead of using `dist/xlsx_html.full.min.js` include latest version of [SheetJS/js-xlsx](https://github.com/SheetJS/js-xlsx/tree/master/dist) along with  `dist/xlsx_html_utils.min.js` and `lib/FileSaver.min.js`.
```
<script type="text/javascript" src="../lib/FileSaver.min.js"></script>
<script type="text/javascript" src="https://raw.githubusercontent.com/SheetJS/js-xlsx/master/dist/xlsx.core.min.js"></script>
<script type="text/javascript" src="../src/xlsx_html_utils.js"></script>
```

Note that the styles will not work with the original version of SheetJS/js-xlsx.

**Comment by gavinhungry** [comment]
@rohithb,

Is there a fork available of [protobi/js-xlsx](https://github.com/protobi/js-xlsx) that this module could use to support styles *and* links?

**Comment by rohithb** [comment]
Released [v1.0.0](https://github.com/linways/table-to-excel/releases/tag/v1.0.0)
Change the backend excel library.
Added support for hyperlink, border colors, exclude rows etc.
